### PR TITLE
Add mewconnect as wallet option

### PR DESCRIPTION
**Pull request by SteveMieskoski**

### CODE REVIEW

**Comment by xiaowheat** [comment]
Could you add a screenshot?

**Comment by SteveMieskoski** [comment]
Yeah, I'll make one.  I'm cleaning up some things with It as well right now.

**Comment by SteveMieskoski** [comment]
Screenshot from 2020-08-20 11-03-19

File: src/modals/ConnectToWalletModal.js [hunk 1]
```diff
@@ -137,6 +151,22 @@ class ConnectToWalletModal extends React.Component {
                   </div>
                 </WalletTypeDiv>
               </Col>
+              <Col span={12}>
```
Comment (xiaowheat) [inline src/modals/ConnectToWalletModal.js:154]: Could we change UI to a row? Currently, it's two rows and the second row has only one wallet icon.

Comment (xiaowheat) [reply src/modals/ConnectToWalletModal.js:154]: Simply change span={12} to span={8}

**Review by xiaowheat — APPROVED** [review]

File: src/modals/ConnectToWalletModal.js (hunk 1 above)
Comment (xiaowheat) [reply src/modals/ConnectToWalletModal.js]: I made a change.

**Pull request merged**